src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -210,24 +224,23 @@ uiModules.get('apps/management')
     loadingCount += 1;
     sendCreateIndexPatternRequest(indexPatterns, {
       id,
+      name,
       timeFieldName,
       notExpandable,
     }).then(createdId => {
       if (!createdId) {
         return;
       }
 
-      refreshKibanaIndex().then(() => {
-        if (!config.get('defaultIndex')) {
-          config.set('defaultIndex', id);
-        }
+      if (!config.get('defaultIndex')) {
```

Comment:
The saved objects API uses `refresh: wait_for` when modifying objects instead of calling the refresh API directly to ensure the consistency.